src/main/java/de/fraunhofer/aisec/codyze/config/CpgConfiguration.kt

```diff
@@ -157,7 +243,30 @@ class TranslationSettings {
                 "If includes is not empty, only the specified files will be parsed and processed in the cpg, unless it is a part of the disabled list, in which it will be ignored. Path must be separated by \':\' (Mac/Linux) or \';\' (Windows)"],
         split = ":|;"
     )
-    var enabledIncludes: Array<File> = emptyArray()
+    fun match(value: Array<File>) {
+        matched = true
+        this.enabledIncludes = value
+    }
+
+    @Option(
+        names = ["--enabled-includes+"],
+        paramLabel = "<path>",
+        description =
+            [
+                "See --enabled-includes, but appends the values to the ones specified in configuration file."],
+        split = ":|;"
```

Comment:
Do we continue using `:|;` or should we switch to system agnostic path separators?